portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,383 @@
-#content {
+body {
+
+    margin: 0px;
+
+}
+
+#random-message {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 100%;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.index-header {
+
+    background-color: rgb(31, 31, 31);
+    display: flex;
+    width: 100%;
+    height: 4.7vw;
+    position: fixed;
+
+}
+
+#index-welcome {
+
+    background-color: rgb(31, 31, 31);
+    display: flex;
+    width: 100%;
+    height: auto;
+
+}
+
+#index-right {
+
+    height: 100%;
+    margin-left: auto;
+    margin-right: 0;
+
+}
+
+#profile-pic {
+
+    max-width: 100%;
+    max-height: 100%;
+    margin-left: auto;
+    margin-right: 0; 
+
+}
+
+#index-left {
+
+    max-width: 100%;
+    max-height: 100%; 
+    margin-bottom: auto;
+    margin-top: auto;
+
+}
+
+#hello-top {
+
+    font-family: "Lucida Console";
+    font-weight: bold;
+    color: ghostwhite;
```

Comment:
It seems like a lot of these styles repeat the color and font-family – one potential issue with this is if you later on decide to change them, you'll have to change it in every location (and it's easy to miss one or two of them).

One way to fix this is to have a class which contains the color you want, and then apply this class to all elements you want to have that color. i.e. you can have a class:

.white-text {
    color: ghostwhite;
}

And then just place the .white-text class in all of these elements, so later on if you change to a different shade of white, you only update the color in one place.

And you can do the same for the font-family which is repeated in many different locations as well (Lucida Console and "Trebuchet MS", Helvetica, sans-serif)

Remember you can use multiple classes for a single HTML element to enable this.